Windows/MainWindow.xaml.cs

```diff
@@ -68,6 +68,7 @@ public partial class MainWindow : Window {
         UserSettingsManager userSettings;
         bool shiftKeyDown;
         bool ctrlKeyDown;
+        double? ctrlScrollGridDivision;
```

Comment:
After giving it some thought, this is no longer needed in this version of the solution and the code can be simplified. I'll refactor it tomorrow.